back-end/models/Teacher.js

```diff
@@ -11,9 +11,21 @@ const teacherSchema = new Schema(
     email: {
       type: String,
     },
+    language: {
+      type: String,
+    },
+    biography: {
+      type: String,
+    },
     phone: {
       type: String,
     },
+    preferences: {
+      type: String,
+    },
+    socialMedias: {
```

Comment:
Teacher can have only one social media? Maybe it's better to hav e an array of strings?